docs/mqtt.md

```diff
@@ -594,6 +594,31 @@ to reconnect to switch to a new virtual host.
 is considered more specific than the port-to-vhost mapping with the `mqtt_port_to_vhost_mapping`
 global parameter and so takes precedence over it.
 
+#### Use client_id from client certificate to authenticate 
+
+By the default, the plugin passes the `client_id` it received from the MQTT connection to the configured authentication backends as a credential. However, it is possible to extract the `client_id` from the certificate in a very similar way as we extract the username. 
+
+First, you must configure which certificate's attribute has the `client_id` by setting the configuration variable `mqtt.ssl_cert_client_id_from`. The allowed values are:
```

Comment:
What does this mean: "ou must configure which certificate's attribute has the `client_id` " does the user need to search which one is it? 

I would say "The acceptable values are:"

Reply:
how about: `you must configure where RabbitMQ extract the client_id from the certificate?`